app/nexus/internal/env/database.go

```diff
@@ -97,6 +97,9 @@ func DatabaseConnMaxLifetimeSec() time.Duration {
 	return time.Hour
 }
 
+// TODO: ensure that all os.Getenv'ed env vars are documented in the readme.
```

Comment:
I know there are a lot of `TODO:`s in the code; I'll take care of them in a follow-up PR.